components/brave_wallet/browser/solana_tx_manager.cc

```diff
@@ -361,6 +366,39 @@ void SolanaTxManager::MakeTokenProgramTransferTxData(
           std::move(callback)));
 }
 
+void SolanaTxManager::MakeTokenProgramTxDataFromMessage(
+    const std::string& message,
+    const mojom::TransactionType tx_type,
+    mojom::SolanaSendTransactionOptionsPtr send_options,
```

Comment:
Is this being used by Jupiter?

Reply:
Yes, Jupiter requires us to set `skipPreflight` to `true`.